pom.xml

```diff
@@ -552,6 +560,33 @@
             </plugins>
         </pluginManagement>
         <plugins>
+            <!-- Maven CI-Friendly Version Management -->
+            <plugin>
```

Comment:
- Where did this CI-friendly view come from? You haven't even submitted a new issue.

Reply:
TLDR, I personally won't agree to merge this PR. First, the next version of elasticjob is 3.1.0, so the master branch is naturally 3.1.0-SNAPSHOT, which is in line with the ASF Infra convention. If you need to compile elasticjob-ui, you have to compile elasticjob first, which will automatically publish the artifacts to the local Maven repository, and elasticjob-ui will extract the JAR from the local repository. Second, shardingsphere itself doesn't use flatten-maven-plugin, and it's strange that its sub-projects need to do so. There are dedicated Maven plugins that can update version numbers in batches. Third, you haven't applied the checkstyle rules of elasticjob for formatting.